keras_nlp/src/utils/transformers/convert.py

```diff
@@ -39,4 +43,8 @@ def load_transformers_tokenizer(cls, preset):
         return load_gemma_tokenizer(cls, preset)
     if cls.__name__ == "Llama3Tokenizer":
         return load_llama3_tokenizer(cls, preset)
-    raise ValueError(f"No conversion huggingface/transformers to {cls}")
+    ValueError(
+        f"{cls} has not been ported from the Hugging Face format yet. "
+        "Please check Hugging Face Hub for the Keras model. "
+        "Models in Keras format should end with `-keras`. (e.g google/gemma-2b-keras)"
```

Comment:
Thanks for updating the message! 

>Models in Keras format should end with `-keras`
I think this naming convention might not be true for every Keras model on HF Hub. So we may want to drop the last line.

Reply:
Sure!